src/Factories/JsonApiErrorFactory.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors\Factories;
+
+use Throwable;
+use WtfPhp\JsonApiErrors\Exceptions\JsonApiErrorException;
+use WtfPhp\JsonApiErrors\Models\JsonApiError;
+
+/**
+ * Class JsonApiErrorFactory
+ * @package WtfPhp\JsonApiErrors\Factories
+ */
+class JsonApiErrorFactory implements JsonApiErrorFactoryInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public static function createFromThrowable(Throwable $throwable): JsonApiError
+    {
+        $jsonError = new JsonApiError();
+        $jsonError->code = $throwable->getCode();
```

Comment:
shouldn't we cast the code to a string here? The json:api specification says "an application-specific error code, expressed as a string value" 🙂